Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1201,4 +1246,479 @@ SPNs have a single credential and most scenarios that use them cannot support mu
+**Properties:** 
+[\*].namespace, [\*].registrationState
+<br />
+
+- Rest API to get Security center pricing tier details for subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Security/pricings?api-version=2023-01-01 <br />
+**Properties:** 
+[\*].name, [\*].properties.pricingTier, [\*].properties.subPlan
+<br />
+<br />
+
+___ 
+
+
+
+## Azure_Subscription_Config_Enable_MicrosoftDefender_ResourceManager
+
+### Display Name 
+Microsoft Defender for Resource Manager should be enabled on subscriptions
+
+### Rationale 
+Microsoft Defender for Resource Manager automatically monitors the resource management operations in your organization. Defender for Cloud detects threats and alerts you about suspicious activity.
+
+### Control Settings 
+```json 
+{
+    "ReqMDCTier": "Standard",
+    "ReqMDCTierResourceTypes": [
+        {
+            "Type": "Arm",
+            "DisplayName": "Resource Manager"
+        }
+    ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> All required resource types are configured with MDC standard tier
+> 
+> **Failed:** 
+> Any of the below condition is satisfied:
+>   - Any of the required resource types is not configured with MDC standard tier.
+>   - MDC Tier is not enabled on the Subscription.
+>   - Fail if security center provider is not registered.
+>   - The access to the security center settings via REST API is denied (ErrorCode such as DisallowedOperations).
+  - The access to the security center settings via REST API is denied (ErrorCode such as DisallowedOperations).
+
+### Recommendation 
+
+- **Azure Portal** 
+    
+    To enable Microsoft Defender for Resource Manager on your subscription: From Defender for Cloud's 'Environment settings' page, select the relevant subscription --> In the 'Defender plans' page, set 'Resource Manager' to 'On'.
+
+
+### Azure Policies or REST APIs used for evaluation 
+
+- API to get providers list with registration status for subscription level : - /subscriptions/{subscriptionId}/providers?api-version=2020-06-01&$select=namespace,registrationstate <br />
+**Properties:** 
+[\*].namespace, [\*].registrationState
+<br />
+
+- Rest API to get Security center pricing tier details for subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Security/pricings?api-version=2023-01-01 <br />
+**Properties:** 
+[\*].name, [\*].properties.pricingTier, [\*].properties.subPlan
+<br />
+<br />
+
+___ 
+
+
+
+## Azure_Subscription_Identity_Rotate_SPN_Credentials
+
+### Display Name 
+App Registrations and Service Principals credentials must be regularly rotated.
+
+### Rationale 
+SPNs having access to subscription must have secrets within maximum approved expiry time.
+
+### Control Settings 
+```json 
+{
+    "ExpirationPeriodInDays": 380,
+    "ServicePrincipalTypeFilter": [ "Application", "Legacy" ],
+    "AllowedObjectIds": []
```

Comment:
remove AllowedObjectIds as it has empty value

Reply:
Fixed